docs/indepth/installation.rst

```diff
@@ -351,9 +356,68 @@ Paste them into a root console on the target node:
 
 .. task:: install_flocker centos-7
    :prompt: [root@node]#
+   
+Finally, you will need to run the ``flocker-ca`` tool that is installed as part of the CLI package.
+This tool generates TLS certificates that are used to identify and authenticate the components of your cluster when they communicate, which you will need to copy over to your nodes. Please see the :ref:`cluster authentication <authentication>` instructions.
```

Comment:
Since this is platform-independent, can we put this in the next section (Cluster Authentication Layer Configuration)?